src/pages/admin/reports/components/pdf-files-section/components/pdf-files-table/PdfFilesTable.module.scss

```
@@ -113,3 +115,102 @@
+
+.rename-cancel {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 24px;
+    height: 24px;
+    padding: 0;
+    border: none;
+    background: none;
+    cursor: pointer;
+    color: c.$error;
```

Stylelint (17.9.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)